frontend/src/lib/taxonomy.tsx

```diff
@@ -230,6 +235,31 @@ export const KEY_MAPPING: KeyMappingInterface = {
             description: 'The initial type of device that was used (first-touch).',
             examples: ['Mobile', 'Tablet', 'Desktop'],
         },
+        $screen_density: {
+            label: 'Screen density',
+            description: 'WHAT IS THE DEFINITION?',
+            examples: [2.75],
+        },
+        $device_model: {
+            label: 'Device Model',
+            description: 'The model of the device that was used.',
+            examples: ['iPhone 12 Pro', 'Pixel 3'],
```

Comment:
```suggestion
            examples: ['iPhone9,3', 'SM-G965W'],
```

See issue https://github.com/PostHog/posthog/issues/18025
The OS returns something like `SM-G965W` instead of the known name which is fine but can be improved.